src/components/TopBar.js

```diff
@@ -266,9 +265,26 @@ export default _.flow(
             ]),
             'Contact Us'
           ]),
-          (trialState === 'Enabled') && enabledCredits,
-          (trialState === 'Enrolled') && enrolledCredits,
-          (trialState === 'Terminated') && terminatedCredits,
+          h(Clickable, {
+            style: styles.nav.supportItem,
+            as: 'a',
+            hover: { backgroundColor: colors.darkBlue[1] },
+            href: 'https://broadinstitute.zendesk.com/hc/en-us',
+            target: '_blank',
+            onClick: () => this.hideNav()
+          }, [
+            div({ style: styles.nav.icon }, [
+              icon('help', {
+                className: 'is-solid',
+                size: 20
+              })
+            ]),
+            'Terra Support',
```

Comment:
Is this what the Comms team or @bradtaylor wanted to name the link?

Reply:
Yes, Brad specified it to be "Terra Support" since he wanted users who needed help (which is what it's called in the mocks) to use the contact us button.